requirements/linting.txt

```diff
@@ -0,0 +1,66 @@
+#
+# This file is autogenerated by pip-compile with python 3.10
+# To update, run:
+#
+#    pip-compile --output-file=requirements/linting.txt requirements/linting.in
```

Comment:
@samuelcolvin JFYK I recommend using the [--resolver backtracking](https://pip-tools.readthedocs.io/en/latest/changelog/#id1) feature of `pip-compile` because it's much more robust at solving complex dependency graphs (boto3 and tensorflow send their regards) and because it's the default resolve in pip now (so using it keeps the results of pip-compile more in line with what you'd get from pip install).